src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -1415,4 +1415,12 @@ Click CANCEL to cancel the update.</value>
   <data name="InstallMessageCaption" xml:space="preserve">
     <value>Install Dynamo</value>
   </data>
+  <data name="DynamoGeometryButtonToolTip" xml:space="preserve">
+    <value>Geometry</value>
+    <comment>Geometry display tooltip</comment>
+  </data>
```

Comment:
This should be renamed to `InCanvasGeomButtonToolTip`, and its content can be rephrased as:

`Enable background 3D preview navigation (Ctrl + G)`